CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.0.54] - 2024-10-03
+
+-   Adds support for using customFramework functions from Node SDK for fullstack frameworks (svelte, remix & astro).
```

Comment:
no need to change version since only boilerplate code is changing, not the actual CLI code.